sentry/src/main/java/io/sentry/CombinedScopeView.java

```diff
@@ -489,4 +489,22 @@ public void setSpanContext(
   public void replaceOptions(@NotNull SentryOptions options) {
     globalScope.replaceOptions(options);
   }
+
+  @Override
+  public @NotNull SentryId getReplayId() {
+    final @NotNull SentryId current = scope.getReplayId();
```

Comment:
@romtsn not quite sure if this is the behaviour we want here

Reply:
after reading the docs, I think it makes sense, as we do it the same way for the other things here!